### PR TITLE
focusing on specific error in server logs

### PR DESCRIPTION
**Pull request by Eskibear**
1. add whitelist of error mssages. Only those on the list will be sent back. Otherwise only tags found in the message will be sent back.
2. After LS service is ready, stop listen to server log changes. So we focus only on startup stage.

Sample Error messages:
- "Error in calling delegate command handler"
- "tags:maven,bundle"
- ""

### CODE REVIEW

File: src/daemon/serverLog/whitelist.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+const TAGS: string[] = [
+    "buildship",
```
Comment (Eskibear) [inline src/daemon/serverLog/whitelist.ts:5]: let me know if you want to add more "keywords"

File: src/daemon/serverLog/whitelist.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    "gradle",
+    "maven",
```
Comment (jdneo) [inline src/daemon/serverLog/whitelist.ts:7]: what about m2e?

Comment (Eskibear) [reply src/daemon/serverLog/whitelist.ts:7]: ok, will add

File: src/daemon/serverLog/whitelist.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    "BadLocationException",
+    "Could not load Gradle version information",
+    "Error computing hover",
+    "Error in calling delegate command handler",
+    "Error in JDT Core during AST creation",
+    "Error occurred while deleting",
+    "Error filtering index locations based on qualifier.",
+    "Failed to publish diagnostics for",
+    "Failed to detect project changes",
+    "Failed to update qualified index.",
```
Comment (testforstephen) [inline src/daemon/serverLog/whitelist.ts:23]: Add one "**Failed to save JDT index**"

Comment (Eskibear) [reply src/daemon/serverLog/whitelist.ts:23]: added.

**Pull request merged**